src/common-sdl/decafsdl_config.cpp

```diff
@@ -266,6 +268,16 @@ loadFrontendToml(std::shared_ptr<cpptoml::table> config)
    }
 
    config::input::vpad0 = config->get_qualified_as<std::string>("input.vpad0").value_or(config::input::vpad0);
+
+   config::input::wpad[0] = config->get_qualified_as<std::string>("input.wpad0").value_or(config::input::wpad[0]);
+   config::input::wpad[1] = config->get_qualified_as<std::string>("input.wpad1").value_or(config::input::wpad[1]);
+   config::input::wpad[2] = config->get_qualified_as<std::string>("input.wpad2").value_or(config::input::wpad[2]);
+   config::input::wpad[3] = config->get_qualified_as<std::string>("input.wpad3").value_or(config::input::wpad[3]);
+
+   config::input::wpadType[0] = config->get_qualified_as<std::string>("input.wpadType0").value_or(config::input::wpadType[0]);
+   config::input::wpadType[1] = config->get_qualified_as<std::string>("input.wpadType1").value_or(config::input::wpadType[1]);
+   config::input::wpadType[2] = config->get_qualified_as<std::string>("input.wpadType2").value_or(config::input::wpadType[2]);
+   config::input::wpadType[3] = config->get_qualified_as<std::string>("input.wpadType3").value_or(config::input::wpadType[3]);
```

Comment:
change these repetitive code to use for loops?

also in other places below and in config.cpp and main.cpp ?

Reply:
It would require using string formatting to do that, probably nicer like it is here?

Reply:
```cpp
   for (int i = 0; i < 4; i++) {
       config::input::wpadType[i] = config->get_qualified_as<std::string>("input.wpadType" + std::to_string(i)).value_or(config::input::wpadType[i]);
   }
```

or simply `std::to_string`, no need to really use anything more fancy like `fmt`. Nothing wrong with some small formatting or use of a lookup table (`std:string[4] digits = {"0", "1", "2", "3"};`).

Reply:
Fair point (👍)